### PR TITLE
Images can now be extracted as .png files if the user adds the -X flag.

### PR DESCRIPTION
**Pull request by nilshermansson**
Not very reliable, still throws exceptions and produces nonsense images sometimes.

### CODE REVIEW

File: pdfminer/image.py [hunk 1]
```diff
@@ -76,26 +76,53 @@ def write_line(self, y, data):
...
 
+class PngWriter:
+    def __init__(self, fp, width, height, color):
+        self.fp = fp
+        self.width = width
+        self.height = height
+        self.color = color
+
+    def write(self, data):
+        from PIL import Image
```
Comment (PrimeNumb) [inline pdfminer/image.py]: Why function-level import over module-level?

**Review by PrimeNumb — COMMENTED** [review]
Missing tests for `PngWriter` and possibly tests for the new option added to pdf2txt

**Review by PrimeNumb — APPROVED** [review]
lgtm 👍

**Review by yang-uu — APPROVED** [review]
lgtm!

**Pull request merged**